pkg/storage/tpr/install_types_test.go

```diff
@@ -175,9 +176,10 @@ func TestInstallTypesPolling(t *testing.T) {
 		t.Fatal("InstallTypes was supposed to error but didn't")
 	}
 
-	for _, name := range getCallArgs {
+	for _, action := range getCallActions {
+		name := action.GetName()
 		if name == serviceBrokerTPR.Name || name == serviceInstanceTPR.Name {
-			t.Errorf("Failed to skip polling for resource that failed to install")
+			t.Error("Failed to skip polling for resource that failed to install", action)
```

Comment:
please use Errorf with %q so "action" stands out from the rest of the words

Reply:
where do you want the `%q`?

Reply:
I put the q at the end.